packages/select/src/Select/components/TriggerButton/TriggerButton.module.scss

```diff
@@ -14,15 +14,17 @@
   flex: 1;
   min-width: 0;
 
-  &:hover .icon,
-  &:global(.story__button-hover) .icon,
-  &:active .icon,
-  &:global(.story__button-active) .icon,
-  &:focus-visible .icon {
-    color: $color-gray-600;
+  &:not(:disabled) {
```

Comment:
Is it possible to not have this? The specificity can sometimes prevent overriding/extending styles sometimes. 

Global selectors are also a little dangerous as they might sneak in and add styles where you don't want them, so a class would be safer :)

Reply:
Hey i replaced the` :disabled` with `.disabled`